andela/mmdp_site
# MMDP-711: Fix inconsistencies in stakeholder directory calculations

**What does this PR do?**

Fixes inconsistencies in stakeholder directory calculations

**description of Task to be completed?**

- Fixes the total no. of beneficiaries on the stakeholder detail screen

 - Fixes amount invested on the stakeholder detail screen

**How should this be manually tested?**

- git checkout `bg-fix-inconsistencies-in-stakeholder-dir-calculations-MMDP-711`

- Start the server, `yarn run:server`

-  On your browser, go to `http://localhost:1234/stakeholder-directory` and click on any stakeholder to view details

- Notice that the `Total number of beneficiaries` and `Amount invested till date` are consistent with what is in the DB

**Any background context you want to provide?**
N/A

**What are the relevant Jira issues?**

[MMDP-711](https://viisaus.atlassian.net/browse/MMDP-711)

**Screenshots (if appropriate)**

![2019-07-23 10 03 20](https://user-images.githubusercontent.com/43179925/61690104-3a9ed000-ad31-11e9-80c2-2054b0c77c17.gif)


**Questions:**

N/A
